projects/extension/src/background/ConnectionManager.ts

```diff
@@ -228,6 +242,18 @@ export class ConnectionManager
     }
   }
 
+  getObjectFromLocalStorage = async (key: string): Promise<string> => {
+    return new Promise((resolve, reject) => {
+      try {
+        chrome.storage.local.get(key, (value) => {
+          resolve(value[key] as string)
+        })
+      } catch (ex) {
+        reject(ex)
+      }
+    })
+  }
```

Comment:
A few things here:
- It should return `Promise<string | undefined>`
- The name is a bit confusing and leaky, what about `getDbContent` instead?
- The `async` is bogus.
- According to [these docs](https://developer.chrome.com/docs/extensions/reference/storage/#usage) we should be doing `get([key], ...` rather than ``get(key, ...`

```suggestion
getDbContent = (key: string): Promise<string | undefined> =>
  new Promise((resolve, reject) => {                        
    try {                                                   
      chrome.storage.local.get([key], (value) => {          
        resolve(value[key])                                 
      })                                                    
    } catch (ex) {                                          
      reject(ex)                                            
    }                                                       
  })                                                        

```